sympy/combinatorics/free_groups.py

```diff
@@ -230,14 +231,40 @@ def index(self, gen):
             raise ValueError("expected a generator of Free Group %s, got %s" % (self, gen))
 
     def order(self):
-        """Returns the order of the free group."""
+        """Returns the order of the free group.
+
+        Examples
+        ========
+
+        >>> from sympy.combinatorics.free_groups import free_group
+        >>> F, x, y = free_group("x, y")
+        >>> F.order()
+        oo
+
+        >>> from sympy.combinatorics.free_groups import free_group
+        >>> F, x, y = free_group("x, y")
+        >>> (x ** 0).order()
+        1
+        """
         if self.rank == 0:
             return 1
         else:
             return S.Infinity
 
     @property
     def elements(self):
+        """
+        Returns the elements of the free group.
```

Comment:
I think, it is preferred that docstrings say "Return" or "Check" rather than "Returns" and "Checks", see [this](https://github.com/sympy/sympy/wiki/Writing-documentation#notes-about-the-summary-section). Though this particular file mostly does "Returns". There's probably no need to change this in the other docstrings but maybe best to say "Return" in the ones you write (not compulsory though).

Reply:
I agree with the consistency and will fix this. Thanks for the note.

Reply:
Consistent with [PEP257](https://www.python.org/dev/peps/pep-0257/): 

>  It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

Reply:
OK. Will fix this now, I probably should have a comprehensive reading on PEP257.